runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    For nvim.                         {Nvim}
+						    Last change:      2015 May
+
+EXTRA KEYMAPS FOR NVIM                                      *nvim-extra-keymaps*
+
+The `plugin/extra-keymaps.vim` script provides a few extra keymaps that might
+be useful and tries to fix some issues the community has identified with vim's
+default keymaps.
+
+The user can disable the plugin by having >
+
+    let g:nvim#defaults#maps#use = 0
+
+in their |vimrc| (additionally, all features can be disabled separately,
+using the variables described below).
+
+============================================================================
+The keymaps provided are:
+
+			 *t_<Esc><Esc>*  *gr:nvim#defaults#maps#leave_term_mode*
+1. Leave terminal-mode (see |:term|) using <Esc><Esc>, besides CTRL-\ CTRL-N
+   (see |CTRL-\_CTRL-N|).
+
+					    *Y-eol* *g:nvim#defaults#maps#Y_eol*
+2. |Y| yanks to the end of line.
```

Comment:
To match the tense used above: `yank to the end of the line`. Same with the other ones.